playground/css/__tests__/css.spec.ts

```diff
@@ -234,11 +234,11 @@ test('less modules composes/from path resolving', async () => {
   // check if the generated CSS module class name is indeed using the
   // format specified in vite.config.js
   expect(await imported.getAttribute('class')).toMatch(
-    /.composed-module__apply-color___[\w-]{5}/,
+    /composed-module__apply-color___[\w-]{5}/,
   )
 
   expect(await imported.getAttribute('class')).toMatch(
-    /.composes-path-resolving-module__path-resolving-less___[\w-]{5}/,
+    /composes-path-resolving-module__path-resolving-less___[\w-]{5}/,
```

Comment:
I removed the `.` from these because they match wild-cards in Regex which are unnecessary here.